test/e2e-test-application/src/app/project/project.component.ts

```diff
@@ -128,6 +128,25 @@ export class ProjectComponent implements OnInit, OnDestroy {
     }
   }
 
+  openAsModal() {
+    linkManager().openAsModal('/projects/pr2/settings', {
+      title: 'microfrontend in a modal',
+      size: 'm',
+    }).then(() => {
+      const newListEl = document.createElement('li');
+      newListEl.textContent = 'promise resolved!';
+      newListEl.id = "promiseResolved";
+      newListEl.className = 'fd-list__item';
+      document.getElementById('promiseTest').appendChild(newListEl);
```

Comment:
this is a little overcomplicated.
I think just one line here: `document.getElementById('promiseTest').innerHTML='promise resolved!' `
would be enough.